modules/abstract-eth/src/abstractEthLikeNewCoins.ts

```diff
@@ -959,7 +959,10 @@ export abstract class AbstractEthLikeNewCoins extends AbstractEthLikeCoin {
       return ethGasConfigs.defaultGasLimit;
     }
     const gasLimitMax = ethGasConfigs.maximumGasLimit;
-    const gasLimitMin = ethGasConfigs.minimumGasLimit;
+    const gasLimitMin =
+      this.staticsCoin?.family === 'opeth' || this.staticsCoin?.family === 'arbeth'
```

Comment:
Why do we need introduce a new default when it can be overriden by passing it?

Reply:
So QA team got this issue, where they set 30000 as gas limit and then the tx failed on-chain. so I thought to add this so that if lower fees are passed, we don't allow. Thoughts?